doc/developer-guides/hld/hld-overview.rst

```diff
@@ -126,20 +128,20 @@ for VM start/stop/pause, virtual CPU pause/resume, etc.
 ACRN 2.0
 ========
 
-ACRN 2.0 is extending ACRN to support pre-launched VM (mainly for safety VM)
+ACRN 2.0 is extending ACRN to support a pre-launched VM (mainly for safety VM)
 and real-time (RT) VM.
 
-:numref:`overview-arch2.0` shows the architecture of ACRN 2.0; the main difference
-compared to ACRN 1.0 is that:
+:numref:`overview-arch2.0` shows the architecture of ACRN 2.0; the main
+differences compared to ACRN 1.0 are that:
 
--  a pre-launched VM is supported in ACRN 2.0, with isolated resources, including
-   CPU, memory, and HW devices, etc.
+-  ACRN 2.0 supports a pre-launched VM, with isolated resources,
+   including CPU, memory, and hardware devices.
 
--  ACRN 2.0 adds a few necessary device emulations in hypervisor like vPCI and vUART to avoid
-   interference between different VMs
+-  ACRN 2.0 adds a few necessary device emulations in the hypervisor, such as
+   vPCI and vUART, to avoid interference between different VMs.
 
--  ACRN 2.0 supports RT VM for a post-launched User VM, with assistant features like LAPIC
-   passthrough and PMD virtio driver
+-  ACRN 2.0 supports an RT VM as a post-launched User VM, with features such as
```

Comment:
I don't think I saw a definition of RT yet, or I missed it.

Reply:
It's in line 131.